data/mods/Tamable_Wildlife/reptile_amphibian.json

```diff
@@ -32,6 +32,63 @@
     "name": { "str": "colossal croaker" },
     "extend": { "flags": [ "CANPLAY", "PET_MOUNTABLE" ] }
   },
+  {
+    "id": "mon_rattlesnake",
+    "type": "MONSTER",
+    "copy-from": "mon_rattlesnake",
```

Comment:
```suggestion
  {
    "id": "mon_rattlesnake_s",
    "type": "MONSTER",
    "copy-from": "mon_rattlesnake_s",
```

Reply:
Looked throught the file, thought I had a duped def, changed it manually.